### PR TITLE
repair base video bug

### PR DESCRIPTION
**Pull request by MasterStreet423**
repair bug "Cannot read properties of undefined (reading 'itemSectionRenderer')" 

[All test passed]

### CODE REVIEW

**Comment by azutake** [comment]
I had a similar problem and used this PR and it improved the situation.
I would like it to be merged!
problem video: https://www.youtube.com/watch?v=_x6MqzkadiM

File: src/youtube/BaseVideo/BaseVideoParser.ts [hunk 1]
```diff
@@ -37,13 +37,11 @@ export class BaseVideoParser {
...
 				?.map((r: YoutubeRawData) => r.text.trim())
-				.filter((t: string) => t) || [];
+				.filter((t: string) => t) || [];	
 		target.description = videoInfo.videoDetails.shortDescription || "";
 
 		// related videos
-		const secondaryContents = data.response.contents.twoColumnWatchNextResults.secondaryResults?.secondaryResults.results.find(
-			(s: YoutubeRawData) => s.itemSectionRenderer
-		).itemSectionRenderer.contents;
+		const secondaryContents = data.response.contents.twoColumnWatchNextResults.secondaryResults?.secondaryResults.results
```
Comment (azutake) [inline src/youtube/BaseVideo/BaseVideoParser.ts:44]:
```suggestion
		const secondaryContents = data.response.contents.twoColumnWatchNextResults.secondaryResults?.secondaryResults.results.find(
			(s: YoutubeRawData) => s.itemSectionRenderer
		)?.itemSectionRenderer?.contents ?? data.response.contents.twoColumnWatchNextResults.secondaryResults?.secondaryResults.results;
```

It seemed necessary to keep what was originally obtainable without change.

Comment (MasterStreet423) [reply src/youtube/BaseVideo/BaseVideoParser.ts:44]: and if I put it inside an if, to check if that element exists, if not, do not do the find

**Comment by SuspiciousLookingOwl** [comment]
Published on `1.6.2`. Sorry for the late merge, I'm currently overseas and didn't have time to check MRs 😅

**Pull request merged**